WPMedia/fusion-news-theme-blocks
# Fixing issues on Identity and Subs blocks

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

- User was being redirected to the wrong URL after asking for a successful Login >> https://arcpublishing.atlassian.net/browse/ASUB-8013
- Nothing was happening after ask for login with Google
- Missing GBP Currency >> https://arcpublishing.atlassian.net/browse/ASUB-8012
- Now I can see Check and GoogleColor Icons as part of themes-components, updating the code accordingly

## Jira Ticket

https://arcpublishing.atlassian.net/browse/ASUB-8013
https://arcpublishing.atlassian.net/browse/ASUB-8012

## Acceptance Criteria

_copy from ticket_

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout ASUB-8013-8012-Login-socialSignIn-icons`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/subscriptions-block,@wpmedia/identity-block`
3. On Subs app you will need to create a price with GBP Currency, and you need to publish this as part of the default offer. 
<img width="1420" alt="Screenshot 2023-11-24 at 12 54 27" src="https://github.com/WPMedia/arc-themes-blocks/assets/54566275/96ed436a-3747-4bda-9af0-4e1508b04c6a">
<img width="1522" alt="Screenshot 2023-11-24 at 12 55 01" src="https://github.com/WPMedia/arc-themes-blocks/assets/54566275/4713bff5-2bd8-4441-b434-a740fdaee4f3">

Once this price is added to the cart you should be able to see the icon corresponding to GBP Currency >> "£"

Ask for login with email+password, the user should be redirected to the URL defined on custom fields
Ask for login with google account, the user should be redirected to the URL defined on custom fields

## Effect Of Changes

![Screenshot 2023-11-24 at 12 50 39](https://github.com/WPMedia/arc-themes-blocks/assets/54566275/d5205ea2-9481-4543-b299-bd0f6464afe0)

Google color icon & GBP 
![Screenshot 2023-11-24 at 12 52 19](https://github.com/WPMedia/arc-themes-blocks/assets/54566275/a52da34a-8d06-499e-943d-89186349e6ca)


### Before

_Example: When I clicked the search button, the button turned red._

[include screenshot or gif or link to video, storybook would be awesome]

### After

_Example: When I clicked the search button, the button turned green._

[include screenshot or gif or link to video, storybook would be awesome]

## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
